code/modules/power/cable.dm

```diff
@@ -488,6 +505,174 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list ( \
 	pixel_y = rand(-2,2)
 	update_icon()
 	recipes = cable_coil_recipes
+	
+/obj/structure/noose //It's a "chair".
+	name = "noose"
+	desc = "Well this just got a whole lot more morbid."
+	icon_state = "noose"
+	buckle_lying = 0
+	icon = 'icons/obj/objects.dmi'
+	anchored = 1
+	can_buckle = 1
+	layer = 5
+	var/image/over = null
+	var/ticks = 0
+
+/obj/structure/noose/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/weapon/wirecutters))
+		user.visible_message("[user] cuts the noose.", "<span class='notice'>You cut the noose.</span>")
+		if(has_buckled_mobs())
+			for(mob/m in buckled_mobs)
+				if(m.mob_has_gravity)
+					m.visible_message("<span class='danger'>[m] falls over and hits the ground!</span>",\
+										"<span class='userdanger'>You fall over and hit the ground!</span>")
+				m.adjustBruteLoss(10)
+		var/obj/item/stack/cable_coil/C = new(get_turf(src))
+		C.amount = 25
+		qdel(src)
+		return
+	..()
+
+/obj/structure/noose/New()
+	..()
+	pixel_y += 16 //Noose looks like it's "hanging" in the air
+	over = image(icon, "noose_overlay")
+	over.layer = MOB_LAYER + 0.1
+
+/obj/structure/noose/Destroy()
+	SSobj.processing.Remove(src)
+	return ..()
+
+/obj/structure/noose/post_buckle_mob(mob/living/M)
+	if(M in buckled_mobs)
+		layer = MOB_LAYER
+		overlays += over
+		SSobj.processing.Add(src)
+		M.pixel_y = initial(M.pixel_y) + 8 //rise them up a bit
+		M.dir = SOUTH
+	else
+		layer = initial(layer)
+		overlays -= over
+		SSobj.processing.Remove(src)

```

Comment:
what if there are still other mobs buckled?, why would you stop processing if only one was unbuckled.